page/page.njk

```diff
@@ -72,9 +72,9 @@
       tag: {
           text: "beta"
       },
-      html: 'This is a new service - your <a class="govuk-link" href="https://www.surveymonkey.co.uk/r/YourFeedbackPB" target="_blank">feedback<span class="govuk-visually-hidden">(opens in new window)</span></a> will help us to improve it.',
+      html: 'This is a new service - your <a class="govuk-link" href="https://www.surveymonkey.co.uk/r/YourFeedbackPB" target="_blank">feedback (opens in new window)<span class="govuk-visually-hidden"></span></a> will help us to improve it.',
```

Comment:
Do we still need the span tag now that we've taken the new window text out of it?